daemon/service/service_nocgo.go

```diff
@@ -0,0 +1,9 @@
+// +build !cgo
+
+// Package service implements the daemon service for mediating access to the
+// storage backend.
+package service
+
+// A dummy implementation of Serve for the environment of Cgo being disabled.
+func Serve(sockpath, dbpath string) {
```

Comment:
Sorry about the confusion. I don't fully understand your previous comments.

From my understanding, you are suggesting that I shall still keep the goroutine as in service.go, and put a `select {}` afterwards which blocks function Serve() till the goroutine receives SIGTERM or SIGINT. Please correct me if I am wrong.